lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -343,6 +315,8 @@ class StandardAccountTest {
         )
         val response = request.send()
 
+        assertNotEquals(Felt.ZERO, response.gasPrice)
+        assertNotEquals(Felt.ZERO, response.gasConsumed)
```

Comment:
I don't think these checks are necessary, esp. the first one considering it's a duplicate